src/control/drpc/rpc.go

```diff
@@ -116,6 +117,7 @@ func (r *Service) ProcessMessage(client *Client, callBytes []byte) ([]byte, erro
 	}
 	respBody, err := module.HandleCall(client, rpcMsg.GetMethod(), rpcMsg.GetBody())
 	if err != nil {
+		log.Debugf("HandleCall for %d:%d failed:%s\n", module.ID(), rpcMsg.GetMethod(), err)
```

Comment:
I think this should be Errorf(), no? We can't rely on Debug logging statements being available in production, and this log entry is the only way for an admin to figure out why a client request failed, right?

Reply:
Errorf makes more sense.